src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Ball {
+    private final Color color;
+    private final int number;
+
+    public Ball(Color color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball with color "
+                + color
+                + ", number "
+                + number
+                + ";";
+    }
+}
```

Comment:
Consider adding an access modifier to the Ball class. Without an explicit access modifier, the class has 'package-private' access by default, which may not be intended. If you want to ensure that the class is accessible everywhere, you should specify it as 'public'.